pkgs/top-level/python-packages.nix

```diff
@@ -18901,6 +18901,8 @@ in {
      };
   };
 
+  proselint = callPackage ../tools/text/proselint { };
```

Comment:
As this is already merged, would you like me to create another pull request? I don't seem to be able to submit further revisions on this one.